traffic_ops/traffic_ops_golang/deliveryservice/letsencryptcert.go

```diff
@@ -374,77 +330,7 @@ func GetLetsEncryptCertificates(cfg *config.Config, req tc.DeliveryServiceLetsEn
-		api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: Added SSL keys with Lets Encrypt", currentUser, logTx)
-		return nil
-	}
-
-	userKeyDer := x509.MarshalPKCS1PrivateKey(userPrivateKey)
-	if userKeyDer == nil {
-		log.Errorf("marshalling private key: nil der")
-		api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: FAILED to add SSL keys with Lets Encrypt", currentUser, logTx)
-		return errors.New("marshalling private key: nil der")
-	}
-	userKeyBuf := bytes.Buffer{}
-	if err := pem.Encode(&userKeyBuf, &pem.Block{Type: "RSA PRIVATE KEY", Bytes: userKeyDer}); err != nil {
-		log.Errorf("pem-encoding private key: " + err.Error())
-		api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: FAILED to add SSL keys with Lets Encrypt", currentUser, logTx)
-		return errors.New("pem-encoding private key: " + err.Error())
-	}
-	userKeyPem := userKeyBuf.Bytes()
-	err = storeLEAccountInfo(userTx, myUser.Email, string(userKeyPem), myUser.Registration.URI)
-	if err != nil {
-		log.Errorf("storing user account info: " + err.Error())
-		api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: FAILED to add SSL keys with Lets Encrypt", currentUser, logTx)
-		return errors.New("storing user account info: " + err.Error())
-	}
-
 	api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: Added SSL keys with Lets Encrypt", currentUser, logTx)
 
 	return nil
 }
-
-func getStoredLetsEncryptInfo(tx *sql.Tx, email string) (*LEInfo, error) {
-	leInfo := LEInfo{}
-	selectQuery := `SELECT email, private_key, uri FROM lets_encrypt_account WHERE email = $1 LIMIT 1`
-	if err := tx.QueryRow(selectQuery, email).Scan(&leInfo.Email, &leInfo.Key, &leInfo.URI); err != nil {
-		if err == sql.ErrNoRows {
-			return nil, nil
-		}
-		return nil, errors.New("getting lets encrypt account record: " + err.Error())
-	}
-
-	decodedKeyBlock, _ := pem.Decode([]byte(leInfo.Key))
-	decodedKey, err := x509.ParsePKCS1PrivateKey(decodedKeyBlock.Bytes)
-	if err != nil {
-		return nil, errors.New("decoding private key for user account")
-	}
-	leInfo.PrivateKey = *decodedKey
-
-	return &leInfo, nil
-}
-
-func storeLEAccountInfo(tx *sql.Tx, email string, privateKey string, uri string) error {
-	q := `INSERT INTO lets_encrypt_account (email, private_key, uri) VALUES ($1, $2, $3)`
-	response, err := tx.Exec(q, email, privateKey, uri)
-	if err != nil {
-		return err
-	}
-
-	rows, err := response.RowsAffected()
-	if err != nil {
-		return err
-	}
-	if rows == 0 {
-		return errors.New("zero rows affected when inserting Let's Encrypt account information")
-	}
-
-	return nil
-}
-
-type LEInfo struct {
-	Email      string `db:"email"`
-	Key        string `db:"private_key"`
-	URI        string `db:"uri"`
-	PrivateKey rsa.PrivateKey
-}
```

Comment:
I don't think you meant to change this file.

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.